pkg/services/ngalert/models/alertmanager.go

```diff
@@ -16,6 +16,7 @@ type AlertConfiguration struct {
 
 // HistoricAlertConfiguration represents a previously used alerting configuration.
 type HistoricAlertConfiguration struct {
+	ID                 int64 `xorm:"pk autoincr 'id'"`
```

Comment:
nit: not sure this field is needed here because HistoricAlertConfiguration extends AlertConfiguration that already declares it.